src/requirements.txt

```diff
@@ -1,21 +1,16 @@
-django~=3.2.16
-djangorestframework~=3.11.0
-django-rest-swagger~=2.2.0
-drf-yasg~=1.17.0
-mozilla-django-oidc~=1.2.3
-whitenoise~=5.0.1
-dj-database-url~=0.5.0
-requests~=2.23.0
-django-watchman~=1.1.0
-django-webpack-loader~=0.6.0
-django-safedelete~=0.5.4
-jsonfield~=3.1.0
+django==3.2.18
+djangorestframework==3.14.0
+drf-spectacular==0.26.0
+mozilla-django-oidc==3.0.0
+whitenoise==6.3.0
+dj-database-url==1.2.0
+requests==2.28.2
+django-watchman==1.3.0
+django-webpack-loader==0.6.0
+django-safedelete==1.3.1
 django-redirect-to-non-www~=0.1.1
-drf-api-tracking~=1.6.0
-django-filter~=22.1
-channels~=2.4.0
-channels-redis~=2.4.2
-twilio~=6.45.4
-
-# Transitive dependency, pinned to fix an error. See: https://github.com/tl-its-umich-edu/remote-office-hours-queue/issues/331
-twisted<21
+drf-api-tracking==1.8.0
```

Comment:
We'll need to keep an eye on this library. It hasn't been updated for a couple years, and doesn't explicitly support Django 3.1 or 3.2.

Reply:
Funny that this was a fork of https://github.com/aschn/drf-tracking. Abandoned projects/forks are still tough to deal with. 

I don't see any forks of this or the other that are more active for this action. So yeah, we'll have to see if anyone else forks or if this gets updates.